extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -336,16 +320,14 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
    */
   populateAdUrlState() {
     // Allow for pub to override height/width via override attribute.
-    const width = Number(this.element.getAttribute('data-override-width')) ||
-      Number(this.element.getAttribute('width'));
-    const height = Number(this.element.getAttribute('data-override-height')) ||
-      Number(this.element.getAttribute('height'));
+    const width = this.getNumberData('overrideWidth') ||
+        Number(this.element.getAttribute('width'));
```

Comment:
Do we know for a fact that window context wouldn't attempt to get width/height from json?